### PR TITLE
Package rename

### PR DESCRIPTION
**Pull request by bitgopatmcl**

### CODE REVIEW

File: packages/express-wrapper/tsconfig.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
       "path": "../io-ts-http"
     },
     {
-      "path": "../superagent-codec-adapter"
+      "path": "../superagent-wrapper"
```
Comment (ericcrosson-bitgo) [inline packages/express-wrapper/tsconfig.json:32]: I presume you made these edits manually? I was writing documentation in https://github.com/typescript-tools/spec this week and pondering the changes required to support npm 7 workspaces without lerna in the `link` command

**Review by ericcrosson-bitgo — APPROVED** [review]

**Comment by ericcrosson-bitgo** [comment]
Rename of packages looks good to me

File: packages/express-wrapper/tsconfig.json (hunk 1 above)
Comment (bitgopatmcl) [reply packages/express-wrapper/tsconfig.json:32]: Yeah, I did it by hand. I wonder if turborepo has a command to dump the dependency graph?

Comment (ericcrosson-bitgo) [reply packages/express-wrapper/tsconfig.json:32]: It does, into `dot`, at least

**Pull request merged**